open-api/swagger.yaml

```diff
@@ -1086,6 +1086,116 @@ paths:
               countsPerStudy:
                 $ref: '#/definitions/CountsMapMap'
 
+  /v2/observations/table:
+    get:
+      description: |
+        Returns a tabular view of observations, as a table using specified dimensions for rows and columns
+      tags:
+        - v2
+      parameters:
+        - name: type
+          required: true
+          in: query
+          type: string
+          description: |
+            The type of data to retrieve. At the moment only 'clinical' is supported. High dimensional
+            data is not supported.
+        - name: constraint
+          required: true
+          in: query
+          type: string
+          description: |
+            JSON-encoded constraint specification. Example: `{"type":"concept","path":"\\Public Studies\\EHR\\Vital
+            Signs\\Heart Rate\\"}`.
+        - name: rowDimensions
+          required: true
+          in: query
+          type: string
+          description: |
+            Json list of strings, with each string being a dimension name. This specifies the dimensions
+            that form the vertical axis of the table (in order). Example: `['study', 'patient']`
+        - name: columnDimensions
```

Comment:
Looking at the description here should be probably `rowDimensions` and the param above `columnDimensions`

Reply:
Not in how I read it. But if you know a better description you're welcome to change it.